goosebit/ui/bff/devices/routes.py

```diff
@@ -31,3 +35,38 @@ def search_filter(search_value):
     total_records = await Device.all().count()
 
     return await BFFDeviceResponse.convert(request, query, search_filter, total_records)
+
+
+@router.patch(
+    "",
+    dependencies=[Security(validate_user_permissions, scopes=["device.write"])],
+)
+async def devices_patch(_: Request, config: DevicesPatchRequest) -> StatusResponse:
```

Comment:
Like in other places: code duplication. Here I don't think we need to expose the patch functionality as external API (yet) - so best to delete it from out official integration API until such a request would come up.

Reply:
Removed from API, but since it was used in tests I had to have the tests use the BFF API.